luigi/execution_summary.py

```diff
@@ -97,7 +101,8 @@ def _get_str(task_dict, extra_indent):
             break
         if len(tasks[0].get_params()) == 0:
             row += '- {0} {1}()'.format(len(tasks), str(task_family))
-        elif _get_len_of_params(tasks[0]) > 60 or (len(tasks) == 2 and len(tasks[0].get_params()) > 1 and (_get_len_of_params(tasks[0]) > 40 or len(str(tasks[0])) > 100)) or len(str(tasks[0])) > 200:
+        elif _get_len_of_params(tasks[0]) > 60 or len(str(tasks[0])) > 200 or \
```

Comment:
I guess there is no more or less pythonic. In my opinion - pythonic mean better readable.

In some cases parenthesis better readable but in this case - this condition `\` looks good. Condition must no be over-parenthesised.


https://www.python.org/dev/peps/pep-0008/#maximum-line-length
> Backslashes may still be appropriate at times. For example, long, multiple with -statements cannot use implicit continuation, so backslashes are acceptable: